automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/Notebook.scala

```diff
@@ -45,6 +48,16 @@ object Notebook extends RestClient with LazyLogging {
     new NotebooksListPage(url + path)
   }
 
+  def createFileAtRootDir(googleProject: GoogleProject, clusterName: ClusterName, fileName: String)(implicit token: AuthToken): File = {
```

Comment:
maybe rename this to something like `createFileInJupyter`?
It was weird to me that creating a file at Root dir requires a http call